.codexdocsrc.sample

```diff
@@ -9,6 +9,7 @@
   "startPage": "",
   "misprintsChatId": "12344564",
   "yandexMetrikaId": "",
+  "twigAppHawkToken": "",
```

Comment:
i think, we need to store both tokens in .env. And also, do not use the "twig" word, because we can change the template framework in a future. Same with the Express.

Better call them "HAWK_TOKEN_CLIENT", "HAWK_TOKEN_BACKEND"